clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileStatus.java

```diff
@@ -12,8 +12,8 @@
  */
 public class LakeFSFileStatus extends FileStatus {
 
-    private String checksum;
-    private String physicalAddress;
+    private final String checksum;
+    private final String physicalAddress;
```

Comment:
Not sure Java value types use final fields. Whatever Hadoop FileStatus does we should copy. (afk...)

Reply:
took the ide recommendations - will revert this file.